Userland/Applications/ThemeEditor/MainWidget.cpp

```diff
@@ -489,7 +489,7 @@ ErrorOr<void> MainWidget::add_property_tab(PropertyTab const& property_tab)
                     TRY(row_widget->load_from_gml(flag_property_gml));
 
                     auto& checkbox = *row_widget->find_descendant_of_type_named<GUI::CheckBox>("checkbox");
-                    checkbox.set_text(String::from_deprecated_string(DeprecatedString(to_string(role))).release_value_but_fixme_should_propagate_errors());
+                    checkbox.set_text(TRY(String::from_utf8(to_string(role))));
```

Comment:
Doesn't StringView have a const char* ctor? I think this diff could've worked before the commit you linked to just as well (?) But LG.

Reply:
StringView's `const char*` ctor was removed a while back, you always have to pass the length or use the literal.